marti_nav_msgs/msg/PlanTrack.msg

```diff
@@ -0,0 +1,7 @@
+# Gives a position and orientation relative to a position on a plan. Published by position trackers.
+
+std_msgs/Header header                      # only stamp used
+
+string plan_id                              # unique ID of the corresponding plan
+geometry_msgs/Pose relative_pose            # pose relative to position on plan
+marti_nav_msgs/PlanPosition plan_position   # position on plan
```

Comment:
couldn't this contents of the PlanPostiion msg be part of this message. Is there a need for two separate messages?

Reply:
shouldn't they be using this PlanTrack message instead to indicate the position along/relative to the planned route?

Reply:
They dont need to the full track, just a index and distance